lightning/src/chain/channelmonitor.rs

```diff
@@ -521,6 +521,56 @@ impl_writeable_tlv_based_enum_upgradable!(ChannelMonitorUpdateStep,
 	},
 );
 
+/// Details about the balance available for claim once the channel appears on chain.
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(test, derive(PartialOrd, Ord))]
+pub enum ClaimableBalance {
+	/// The channel is not yet closed (or the initial commitment or closing transaction has not yet
+	/// been confirmed). The given balance is claimable (less on-chain fees) if the channel is
```

Comment:
Hmmm confusing, just say "the channel is not yet closed, either cooperatively or non-cooperatively" ? I don't see what the distinctions initial commitment/beyond-initial commitment/closing bring.

Reply:
Is it better now? I think its very important we note that we will still use this before the commitment appears in a block, even if the channel is "closed" from the users' perspective.